pxtlib/cloud.ts

```diff
@@ -1,4 +1,19 @@
 namespace pxt.cloud {
+    export type DevBackendType = "default" | "prod" | "staging" | "localhost";
+
+    const DEV_BACKEND_PROD = "https://www.makecode.com";
+    const DEV_BACKEND_STAGING = "https://staging.pxt.io";
+    const DEV_BACKEND_LOCALHOST = "http://localhost:8080";
+
+    export const DEV_BACKEND = DEV_BACKEND_PROD;
```

Comment:
nit; if this is being exported we might as well give it a proper type to avoid having to `<any>` like below / false errors that might show up elsewhere when it's type is super narrowed by the const; e.g.

```typescript
type BackendUrls = typeof DEV_BACKEND_PROD | typeof DEV_BACKEND_STAGING | typeof DEV_BACKEND_LOCALHOST;
export const DEV_BACKEND: BackendUrls = DEV_BACKEND_PROD;

export function devBackendType(): DevBackendType {
    if (DEV_BACKEND === DEV_BACKEND_PROD) return "prod";
    if (DEV_BACKEND === DEV_BACKEND_STAGING) return "staging";
    if (DEV_BACKEND === DEV_BACKEND_LOCALHOST) return "localhost";
    return "prod";
}
```